forch/forchestrator.py

```diff
@@ -138,28 +141,40 @@ def _process_static_device_behavior(self):
         for mac, device_behavior in devices_state.device_mac_behaviors.items():
             self._process_device_behavior(mac, device_behavior, static=True)
 
-    def _calculate_behavioral_config(self):
-        behavioral_config_file = self._config.get('orchestration', {}).get('behavioral_config_file')
-        if behavioral_config_file:
-            self._behavioral_config_file = os.path.join(
-                os.getenv('FAUCET_CONFIG_DIR'), behavioral_config_file)
-            return True
+    def _calculate_config_files(self):
+        orchestration_config = self._config.get('orchestration', {})
 
+        behavioral_config_file = orchestration_config.get(
+            'behavioral_config_file', _BEHAVIORAL_CONFIG_DEFAULT)
```

Comment:
shouldn't this defailt to the file as specified by the ENV variable?

Reply:
It seems that there is no ENV for Faucet config file set for Forch. Do you mean the one used by Faucet in production?